Content.Server/Chemistry/Metabolism/HealthChangeMetabolism.cs

```diff
@@ -0,0 +1,48 @@
+using Content.Server.GameObjects.Components.Nutrition;
+using Content.Shared.Chemistry;
+using Content.Shared.Damage;
+using Content.Shared.GameObjects.Components.Damage;
+using Content.Shared.Interfaces.Chemistry;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Serialization;
+using Robust.Shared.Serialization;
+
+namespace Content.Server.Chemistry.Metabolism
+{
+    /// <summary>
+    /// Default metabolism for medicine reagents. Attempts to find a DamegableComponent on the target,
+    /// and to update it's damage values.
+    /// </summary>
+    class HealthChangeMetabolism : IMetabolizable
```

Comment:
IMO better off calling it DamageMetabolism given its dual purpose.

Also didn't this not work unless it was public last time?